align/pnr/toplevel.py

```diff
@@ -88,14 +88,15 @@ def RouteWork( mode, current_node, *, metal_l=signal_routing_metal_l, metal_u=si
             logger.info("End MNA")
             #return
 
-
-        RouteWork(2, current_node, metal_l=power_grid_metal_l, metal_u=power_grid_metal_u)
+        if len(current_node.PowerNets)>0:
```

Comment:
The Pythonic way to write this is:
```
if current_node.PowerNets:
```

Reply:
@stevenmburns 

I see. Thank you.

Will change to pythonic way.